zebra-network/src/lib.rs

```diff
@@ -12,6 +12,10 @@
 //! [`tower::Service`] representing "the network" which load-balances
 //! outbound [`Request`]s over available peers.
 //!
+//! Unlike the underlying legacy network protocol the provided
+//! [`tower::Service`] guaruntees that each `Request` future will resolve to
```

Comment:
```suggestion
//! [`tower::Service`] guarantees that each `Request` future will resolve to
```

Reply:
I swear to god this word is my arch nemesis. When I wrote this I was like, "did I spell it wrong this time? naw, looks fine. :smiley:"

NOPE